runtime/v2/builder.go

```diff
@@ -119,6 +124,30 @@ func (a *AppBuilder[T]) Build(opts ...AppBuilderOption[T]) (*App[T], error) {
 	}
 	a.app.stf = stf
 
+	v := a.viper
+	home := v.GetString(FlagHome)
+
+	storeOpts := rootstore.Options{}
```

Comment:
Let's move `serverv2store.DefaultStoreOptions` to store and use it here too (so we have defaults set just in case)